genie-core/src/main/java/com/netflix/genie/core/jpa/entities/ApplicationEntity.java

```diff
@@ -50,9 +47,10 @@
  */
 @Getter
 @Setter
+@ToString(callSuper = true)
```

Comment:
A `toString()` or the call to super? I added the `toString()` because we tend to log things out a lot so having a nice toString is better than the default one from Java.

Calling super to make sure the members of parent classes are included.

Reply:
Do you want all the members to be included? It might end up querying all the lazy associations too just for logging.

Reply:
Good point @ajoymajumdar. I'll add exclusions to the collections